app/debrief_svc.py

```diff
@@ -1,4 +1,7 @@
 import logging
+from datetime import timedelta
+
+lateral_movement_tolerance = timedelta(minutes=1)
```

Comment:
I'd probably ALL_CAPS this name to make it pop if you leave it as a module level constant.  Though also consider making it a DebriefService instance attribute (easy for the caller to change when instantiating the service): 

```python
def __init__(self, services, lateral_movement_tolerance_minutes=1):
    ...
    self.lateral_movement_tolerance = timedelta(minutes=lateral_movement_tolerance_minutes)
```

Reply:
added as an instance attribute in most recent commit